src/nunit.analyzers/TestCaseUsage/TestCaseUsageAnalyzer.cs

```diff
@@ -43,21 +43,26 @@ public override void Initialize(AnalysisContext context)
         {
             context.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.None);
             context.EnableConcurrentExecution();
-            context.RegisterSymbolAction(TestCaseUsageAnalyzer.AnalyzeMethod, SymbolKind.Method);
+            context.RegisterCompilationStartAction(AnalyzeCompilation);
         }
 
-        private static void AnalyzeMethod(SymbolAnalysisContext context)
+        private static void AnalyzeCompilation(CompilationStartAnalysisContext context)
```

Comment:
Please rename to `AnalyzeCompilationStart`

Reply:
```suggestion
        private static void AnalyzeCompilationStart(CompilationStartAnalysisContext context)
```